ledger/light/pulsemanager/pulsemanager.go

```diff
@@ -266,7 +266,7 @@ func (m *PulseManager) getExecutorHotData(
 		return nil, err
 	}
 
-	hotIndexes := make([]message.HotIndex, len(bucks))
+	hotIndexes := []message.HotIndex{}
```

Comment:
Might be it's better like that :
hotIndexes := make([]message.HotIndex, 0, len(bucks))
?
To prevent reallocation due to consequence "append"s